api/envoy/config/bootstrap/v3/bootstrap.proto

```diff
@@ -101,6 +101,17 @@ message Bootstrap {
     core.v3.ApiConfigSource ads_config = 3;
   }
 
+  message ApplicationLogFormat {
```

Comment:
Would it make sense to rename the message to something more generic, like `AppicationLog`? 

There might be need for other application log related configuration in the (near) future. I've been working on https://github.com/envoyproxy/envoy/pull/27579 (design doc to-be-written) and I likely need something in bootstrap to configure application log masking rules. So maybe it would be nice to have common message to avoid adding many top-level application log related fields?

Cc @yanavlasov

Reply:
Ack - I nested the log format config in ApplicationLogConfig message. Thanks for that masking work - seems important.